ArduCopter/GCS_Mavlink.cpp

```diff
@@ -809,9 +809,8 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
             } else {
                 copter.wp_nav->set_speed_xy(packet.param2 * 100.0f);
             }
-            return MAV_RESULT_ACCEPTED;
         }
-        return MAV_RESULT_FAILED;
+        return MAV_RESULT_ACCEPTED;
```

Comment:
This is incorrect according to the spec. `-1` is no change - other negative values should mean failure.

Pretty sure we don't want to bring in any changes like this until 4.1 is out the door.